bin/templates/project/app/build.gradle

```diff
@@ -103,10 +111,12 @@ if (hasBuildExtras2) {
 }
 
 // Set property defaults after extension .gradle files.
-if (ext.cdvCompileSdkVersion == null) {
-    ext.cdvCompileSdkVersion = privateHelpers.getProjectTarget()
-    //ext.cdvCompileSdkVersion = project.ext.defaultCompileSdkVersion
-}
+ext.cdvCompileSdkVersion = cdvCompileSdkVersion == null ? (
```

Comment:
`compileSdkVersion` should probably default to `targetSdkVersion`, unless it is explicitly set